content/en/docs/releases/next-release-preview/index.md

```diff
@@ -10,3 +10,15 @@ in the next release of Spinnaker. These notes will be prepended to the release
 changelog.
 
 ## Coming Soon in Release 1.30
+
+### Spring Boot 2.3
+
+Spinnaker 1.30 uses spring boot 2.4, where 1.29 uses spring boot 2.3.  Spring
+boot 2.4 considers session data cached by spring boot 2.3 invalid.  Therefore,
+users with cached sessions will be unable to log in until the invalid
+information is removed from the cache.  Open browser windows to spinnaker are
+unresponsive after the deployment until they’re reloaded.  Executing:
+
+    $ redis-cli keys "spring:session*" | xargs redis-cli del
+
+on gate's redis instance removes the cached session information.
```

Comment:
Couple of caps missing :)

s/spring boot/Spring Boot
s/gate/Gate
s/spinnaker/Spinnaker

Reply:
Capitalizing spring boot and spinnaker makes sense.  I'm less sure about gate, but going for it....Lemme change similar text in the 1.29 changelog at the same time: https://github.com/spinnaker/spinnaker.io/pull/253/commits/f20fabd3e699aa5f8d4cd4b032c527d0df0cdfec